src/pages/plan-trip/plan-trip.html

```diff
@@ -30,7 +30,10 @@
         type="text" placeholder="Enter a destination" [(ngModel)]="request?.destination.name"/>
       </label>
     </ion-item>
-    <ion-item>
+    <ion-item *ngIf="isInternetExplorer">
+      <p style="font-style: italic;">Internet Explorer only supports finding buses leaving now.</p>
```

Comment:
Slight grammatical ambiguity: the buses are leaving now, but this sentence could also be read to imply that IE now only supports finding leaving buses.

"buses **that are** leaving now", perhaps?